deployment/GI03FLMA_Deploy.csv

```diff
@@ -207,55 +207,55 @@ AR60-01,,AR69-01,,GI03FLMA-RIS01-00-SIOENG000,8,1,2021-08-13T13:22:00,2022-07-08
+AR76-03,,,,GI03FLMA-RI000-00-VELPTB100,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-VELPTB-11970,59.7711,-39.883083,,2591,2691,,,C
+AR76-03,,,,GI03FLMA-RI000-00-VELPTB400,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-VELPTB-12236,59.7711,-39.883083,,2291,2691,,,C
+AR76-03,,,,GI03FLMA-RI000-00-VELPTB700,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-VELPTB-12652,59.7711,-39.883083,,1991,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-00-SIOENG000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGCON-SIOCON-14883,59.7711,-39.883083,,1507,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-ADCPSL003,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-ADCPSL-21446,59.7711,-39.883083,,500,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG040,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-11608,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG041,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13427,59.7711,-39.883083,,40,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG042,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13599,59.7711,-39.883083,,60,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG043,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-11606,59.7711,-39.883083,,90,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG044,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13594,59.7711,-39.883083,,130,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG045,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13419,59.7711,-39.883083,,180,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG046,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13622,59.7711,-39.883083,,250,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG047,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13601,59.7711,-39.883083,,350,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG048,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13590,59.7711,-39.883083,,500,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH049,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13633,59.7711,-39.883083,,750,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH050,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-12219,59.7711,-39.883083,,1000,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH051,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13447,59.7711,-39.883083,,1500,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-00-SIOENG000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGCON-SIOCON-13283,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-03-DOSTAD000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-DOSTAD-00224,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-04-PHSENF000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-PHSENF-P0141,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-05-FLORTD000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-FLORTD-01299,59.7711,-39.883083,,30,2691,,,C
```

Comment:
Looks good

Reply:
new midwater and anchor recovery times look good for flma9. deployment time for flma10 looks good. ctdmoh updates look good based on what is in the config sheet for flma10.